README.md

```diff
@@ -303,6 +303,109 @@ The credit card charge result may contains `redirect_url` for 3DS authentication
 For full example on Credit Card 3DS transaction refer to:
 - [Flask App examples](/examples/flask_app) that implement Snap & Core Api
 
+### 2.2.D Subscription API
+You can see some Subscription API examples [here](examples/subscription), [Subscription API Docs](https://api-docs.midtrans.com/#subscription-api)
+
+Subscription API only support for `Credit Card` and `Gopay`
```

Comment:
No need to include this info, if other new payment methods support subscription this information can become invalid.